src/ggrc-client/js/components/unified-mapper/mapper-results.js

```diff
@@ -384,6 +384,7 @@ export default can.Component.extend({
             self.isDisabled = true;
             if (isMegaMapping) {
               self.mapAsChild = null;
+              self.isMega = true;
```

Comment:
Nice approach! Let's make a few renamings:
```suggestion
              self.isSelf = true;
```
as it mostly states that the item in the list is the baseInstance itself.

Reply:
done